daemon/daemon_test.go

```diff
@@ -42,6 +43,16 @@ func TestDaemonInstallDriver(t *testing.T) {
 	require.Nil(err)
 }
 
+func TestDaemon_InstallNonexistentDriver(t *testing.T) {
+	require := require.New(t)
+	s, tmp := buildMockedDaemon(t)
+	defer os.RemoveAll(tmp)
+
+	err := s.InstallDriver("", "docker://list", false)
+	require.Error(err, "An error was expected")
```

Comment:
```suggestion
	require.NotNil(err, "An error was expected")
```

Reply:
Could you please elaborate on this suggestion, to help understand it better? 

From [require.Error GoDoc](https://godoc.org/github.com/stretchr/testify/require#Error)
```
Error asserts that a function returned an error (i.e. not `nil`).
```

Reply:
Sorry, confused it with a similar function from a different package :)